orchestrators/FinanceOperations/tests/Unit/DataProviders/TreasuryDataProviderTest.php

```
@@ -0,0 +1,157 @@
+
+    public function testGetCashFlowForecastAggregatesInflowsAndOutflows(): void
+    {
+        $receivableQuery = $this->createMock(ReceivableQueryInterface::class);
+        $payableQuery = $this->createMock(PayableQueryInterface::class);
+
+        $receivableQuery
+            ->expects($this->once())
+            ->method('getExpectedReceipts')
+            ->willReturn([
+                new class {
+                    public function getExpectedDate(): \DateTimeImmutable { return new \DateTimeImmutable('2026-05-01'); }
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '73', column '88'). (undefined)

(MissingImport)